lib/ffe.gd

```diff
@@ -668,7 +668,7 @@ DeclareAttribute( "AsInternalFFE", IsFFE);
 ##
 ##  <Description>
 ##  <Ref Func="RootFFE"/> returns a finite field element 
-##  <A>r</A> from <F> whose <A>k</A>-th power is <A>z</A>.
+##  <A>r</A> from <A>F</A> whose <A>k</A>-th power is <A>z</A>.
```

Comment:
The documentation for `RootFFE` is, however, incomplete:  `F` is optional, and one can also instead pass in the size of the finite field (i.e., a prime power). Whether we want to document that or not is another question....?

Reply:
I think the initial use was for the size of the field and we then realized that we need to allow different bases. I think both uses have their place, and I thus would suggest we document them both.